README.md

```diff
@@ -334,7 +316,110 @@ You can access a vector of pointers to the parsed options in the original order
 If `--` is present in the command line that does not end an unlimited option, then
 everything after that is positional only.
 
-#### Getting results 🚧
+#### Validators
```

Comment:
TOC needs change too?

Reply:
I think the Readme needs another pass especially with the option_group merge, so I can double check to make sure the TOC matches the headers,  Would you want all the subtitles in the TOC?

Reply:
Unless it makes it too long, I think it would be fine.